electron/electron.ffmpeg.js

```diff
@@ -0,0 +1,83 @@
+/*
+    FFMpeg wrapper
+*/
+
+import log from 'electron-log';
+import { spawn } from 'child_process';
+import { main } from '../config/config';
+
+const { rtmpPort, frameConfig } = main;
+
+/* track if user stopped the process */
+let userStopFFmpeg = false;
+
+const startFFMpeg = (sender, rtmpStrmID, configIdx = 0) => {
+    log.info(`Launching FFmpeg with config: ${configIdx}`)
+    userStopFFmpeg = false  /* reset !!*/
+
+    const framerate = frameConfig[configIdx].framerate
+    const keyint = frameConfig[configIdx].keyint
+    const FFMPeArgs = [
+        '-f', 'avfoundation',
+        '-framerate', framerate,
+        '-pixel_format', 'uyvy422',
+        '-i', '0:0',
+        '-vcodec', 'libx264',
+        '-tune', 'zerolatency',
+        '-b', '900k',
+        '-x264-params', `keyint=${keyint}:min-keyint=${keyint}`,
+        '-acodec', 'aac',
+        '-ac', '1',
+        '-b:a', '96k',
+        '-f', 'flv',
+        `rtmp://localhost:${rtmpPort}/stream/${rtmpStrmID}`];
+
```

Comment:
Are you running it within the context of Electron or directly from command line?  If it's the second, it's a problem with the input devices - the error message is saying your default camera or mic is not working.  If it's the first, I'm guessing it's some kind of permission issue.

Btw, I've also seen this error happen before with an old terminal session - restarting my terminal solved it, and I have no idea why.